Makefile.am

```diff
@@ -162,7 +162,7 @@ endif
 	@echo "Remember you must run ldconfig before using the above settings"
 	@echo "--------------------------------------------------------------"
 	$(MKDIR_P) $(DESTDIR)$(lockdir) $(DESTDIR)$(logdir)
-	$(CHGRP) pkcs11 $(DESTDIR)$(lockdir) $(DESTDIR)$(logdir)
+	$(CHGRP) $(pkcs_group) $(DESTDIR)$(lockdir) $(DESTDIR)$(logdir)
```

Comment:
Below in line 175 and 176 is pkcs11 hard coded. Should it also be $(pkcs_group)?

Reply:
In lines 175 and 176 it is the name of a directory that must always be named 'pkcs11', and has nothing to do with the group name.